pkgs/development/python-modules/mdutils/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "mdutils";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "didix21";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-regIrMWbGmW574dfojxZFJoivpaqOpN1I6YsqLEp8BM=";
+  };
+
+  checkPhase = ''
+    python -m unittest discover
+  '';
```

Comment:
The overridden phase needs the pre and post hooks.

```nix
  checkPhase = ''
    runHook preCheck
    ${python.interpreter} -m unittest discover
    runHook postCheck
  '';
```

Reply:
I was seeing `${python.interpreter}` being used in other Python modules. Why? Isn't it inherited from `buildPythonPackage` as-is?

Reply:
> Why?

Pypy's binaries is not called python.

Reply:
Makes sense. I completely forgot about PyPy.